Makefile

```diff
@@ -2,14 +2,18 @@
 # with Go source code. If you know what GOPATH is then you probably
 # don't need to bother with make.
 
-.PHONY: geth mist all test travis-test-with-coverage clean
+.PHONY: geth evm mist all test travis-test-with-coverage clean
 GOBIN = build/bin
 
 geth:
-	build/env.sh go install -v $(shell build/ldflags.sh) ./cmd/geth
+	build/env.sh $(GOROOT)/bin/go install -v $(shell build/ldflags.sh) ./cmd/geth
```

Comment:
GOROOT is not always present in the environment.Using it here would require that people set it before building.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          ‎